drivers/flash/nrf_qspi_nor.c

```diff
@@ -487,16 +487,19 @@ static inline int qspi_nor_read_id(struct device *dev,
 static int qspi_nor_read(struct device *dev, off_t addr, void *dest,
 			 size_t size)
 {
+	void *dptr = dest;
+	size_t dlen = size;
+	u8_t buf[4];
+
 	if (!dest) {
 		return -EINVAL;
 	}
 
 	/* read size must be non-zero multiple of 4 bytes */
-	if (((size % 4U) != 0) || (size == 0)) {
-		return -EINVAL;
-	}
-	/* address must be 4-byte aligned */
-	if ((addr % 4U) != 0) {
+	if (size < 4U) {
```

Comment:
Can you throw away the comment from line above or change it to something that describes the operation. I know it is not yours but it is basically type of "Assigning 0 to i" comment.

Reply:
Updated comment now. Hope it's better.

Reply:
Thank!